desktop/package.json

```diff
@@ -39,17 +39,17 @@
     "getenv": "0.6.0",
     "immutable": "3.8.1",
     "jsonfile": "2.3.1",
-    "lodash": "4.13.1",
-    "marked": "0.3.5",
-    "material-ui": "0.15.2",
-    "menubar": "5.0.0",
+    "lodash": "4.14.2",
+    "marked": "0.3.6",
+    "material-ui": "0.15.3",
+    "menubar": "5.1.0",
```

Comment:
this is an electron issue so we'd have to do it there. this is better than what happens w/ the version we have imho